examples/people-to-csv.py

```diff
@@ -0,0 +1,25 @@
+"""Example to scrape a list of Companies, and put overviews in csv form"""
+
+from scrape_linkedin import CompanyScraper
+import pandas as pd
+
+# LIST YOUR COMPANIES HERE
+my_company_list = [
+    'facebook', 'mit-sloan-school-of-management', 'linkedin',
+    'harvard-university'
+]
+
+company_data = []
+
+with CompanyScraper() as scraper:
+    # Get each company's overview, add to company_data list
+    for name in my_company_list:
+        sc = scraper.scrape(company=name, people=True)
```

Comment:
optional: As a naive reader it would be unclear to be what `sc` is supposed to be. I would suggest: `company_info`, or similar.